udata_front/theme/gouvfr/datagouv-components/src/components/DataserviceLinks/DataserviceLinks.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div>
+        <div 
+            class="border border-blue-400 fr-p-2w fr-dataservice__box"
```

Comment:
```suggestion
            class="border border-blue-400 fr-p-2w bg-alt-grey"
```

The existing bg-alt-grey class should be used instead of adding a new fr-dataservice__box